packages/services/src/vckit.service.ts

```diff
@@ -43,18 +51,58 @@ export const issueVC = async ({
   type,
   issuer,
   credentialSubject,
+  credentialStatus,
   restOfVC,
   vcKitAPIUrl,
 }: IVcKitIssueVC): Promise<VerifiableCredential> => {
-  const body = constructCredentialObject({ context, type, issuer, credentialSubject, ...restOfVC });
-  privateAPI.setBearerTokenAuthorizationHeaders(appConfig.defaultVerificationServiceLink.apiKey ?? '');
+  const apiKey = appConfig.defaultVerificationServiceLink.apiKey ?? '';
+  let _credentialStatus = credentialStatus ? { ...credentialStatus } : null;
+
+  // issue credential status if not provided
+  if (!_credentialStatus) {
```

Comment:
That's a good idea! I'll update it.

Reply:
Based on my conversation with @namhoang1604, I will update the `issueCredentialStatus` function to include additional validation and provide more meaningful error messages. I believe this function shouldn’t be wrapped in a try-catch block, as it will be used in a context that already has a try-catch block in place.